style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  align-items: stretch;
+  border-radius: 3px;
+  text-align: center;
+}
+
+footer a {
+  text-decoration: none;
+  color: #6f93d0;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  color: white;
+}
+
+.nav-bar div {
+  width: 70px;
+  background:
+    linear-gradient(
+      to top,
+      #4f4f4f 0%,
+      #393939 50%,
+      #444 50%,
+      #606060 100%
+    );
+  color: white;
+  height: 25px;
+  border-right-style: solid;
+  border-right-color: grey;
+}
+
+#searchbutton {
+  background:
+    linear-gradient(
+      to top,
+      #4f4f4f 0%,
+      #393939 50%,
+      #444 50%,
+      #606060 100%
+    );
+  height: 20px;
+}
+
+.search {
+  position: relative;
+  height: 100%;
+}
+
+.fa-search {
+  position: absolute;
+  left: 5px;
+  bottom: 3px;
+  color: white;
+}
+
+header {
+  background: url("./images/apple2.jpg");
+  background-repeat: no-repeat;
+  min-height: 500px;
+  background-size: 100% auto;
```

Comment:
- [ ] Change background-size to 'cover'. This way the image will always occupy all of its backgrounds.